### PR TITLE
Feature/accordion component

### LINKED ISSUE #46: [FEATURE] Create Accordion Component
## Description
Develop an Accordion Component. This component will be implemented in the pricing page to list all FAQs. The Accordion component should not drop down more than 1 item.

## Acceptance Criteria

- The accordion component should be implemented according to the design [Use this as guideline](https://ui.shadcn.com/docs/components/accordion)
- Ensure the accordion component is responsive in all screen sizes
- Write test for your component

### Requirements
**The prop should receive the Accordion Header, the question, and the Accordion content, the response**

### Test Script

- Ensure the component renders the requirements accordingly
- Ensure the component receives the accordion header, the question.
- Ensure the component receives the accordion content, the response.

## Tasks

- [x] Develop the component based on the requirements and design
- [x] Make component responsive on all screens
- [x] Write unit test for the component

### Link
[Accordion](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=4-20796&m=dev)

### PR DESCRIPTION
**Pull request by Confidenceb**
# Description

**Closes #46 **

# Changes proposed

## What did you do?

The Accordion Component has been developed and implemented for the pricing page to list all FAQs. This component ensures that no more than one item is expanded at a time, as per the acceptance criteria.

Dependencies installed: 
	Eslint Dependencies
	pnpm install
	Vitest Dependencies

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [] My code follows the code style of this project.
- [] This PR does not contain plagiarized content.
- [] The title and description of the PR is clear and explains the approach.
- [] I am making a pull request against the **dev branch** (left side).
- [] My commit messages styles matches our requested structure.
- [] My code additions will fail neither code linting checks nor unit test.
- [] I am only making changes to files I was requested to.

# Screenshots/Videos

image
image
image

### CODE REVIEW

File: .vscode/extensions.json
Comment (prudentbird) [inline .vscode/extensions.json:1]: why did you change this file?

File: .vscode/settings.json
Comment (prudentbird) [inline .vscode/settings.json:1]: Why did you change this file??

File: components.json
Comment (prudentbird) [inline components.json:1]: Why did you change this file??

File: eslint.config.mjs
Comment (prudentbird) [inline eslint.config.mjs:1]: Why did you change this file??

File: package.json
Comment (prudentbird) [inline package.json:1]: Why did you install packages?

**Review by prudentbird — COMMENTED** [review]
Go and resolve your merge conflicts!!!!!

File: package.json
Comment (Confidenceb) [reply package.json:1]: I was trying to test and is not working
I'm trying to resolve it now

**Comment by oreoluwa212** [comment]
You need to fetch from upstream, use the globals, check the boxes here that you have completed, run lints and test (attached the screenshots)

Nothing pASSES ON THIS PR

**Comment by Confidenceb** [comment]
I've checked the boxes
